pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -22,6 +22,21 @@ const (
 	auditPolicyAsset = "pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml"
 )
 
+// DefaultPolicy brings back the default.yaml audit policy to init the api
+func DefaultPolicy() ([]byte, error) {
+	cm, err := GetAuditPolicies("kube-apiserver-audit-policies", "openshift-kube-apiserver")
```

Comment:
```
cm, err := GetAuditPolicies("kube-apiserver-audit-policies", "openshift-kube-apiserver")
```

this makes it specific to kas operator, in library-go we usually put logic that can be shared by all operators. My original suggestion was to pass the name and namespace as parameters:
```
func GetDefaultPolicy(name, namespace string) ([]byte, error) {
   cm, err := GetAuditPolicies(name, namespace)
}
```